lib/AArch64Lexer.mli

```diff
@@ -18,4 +18,5 @@
 
 module Make : functor(O:LexUtils.Config) -> sig
   val token : Lexing.lexbuf -> AArch64Parser.token
+  val check_name : string -> AArch64Parser.token
```

Comment:
I am wondering, why export check_name ?

Reply:
Later I shall use it to parse assembly using a different grammar parsed to Menhir. 
Details of how Menhir does this comes under _partial specifications_ in the Menhir manual http://gallium.inria.fr/~fpottier/menhir/manual.pdf

Reply:
ie this allows us to reuse assembly mnemonics elsewhere